packages/storefront-api-client/src/tests/storefront-api-client/storefront-api-client.test.ts

```diff
@@ -34,6 +34,7 @@ describe("Storefront API Client", () => {
     const mockRequestResponse = {
       data: {},
     };
+    const mockRequestStreamResponse = {};
```

Comment:
No - in these tests, we're only testing that the SFAPI client is just returning the same thing the underlying `graphQL client` is returning for each of the request/fetch functions since they're straight up passthroughs.